Mathlib/Data/ENat/Basic.lean

```diff
@@ -313,15 +313,19 @@ lemma add_lt_add_iff_left {k : ℕ∞} (h : k ≠ ⊤) : k + n < k + m ↔ n < m
 
 lemma ne_top_iff_exists : n ≠ ⊤ ↔ ∃ m : ℕ, ↑m = n := WithTop.ne_top_iff_exists
 
-lemma eq_top_iff_forall_ne : (∀ m : ℕ, ↑m ≠ n) ↔ n = ⊤ := WithTop.forall_ne_iff_eq_top
```

Comment:
It looks to me like the lemma `eq_top_iff_forall_ne :  n = ⊤ ↔ ...` should exist, since `ne_top_iff_exists` does: what do you think?

Reply:
Sure, I don't have strong opinions here